microsite/src/pages/plugins/plugins.module.scss

```diff
@@ -28,10 +28,57 @@
   :global(.pluginsContainer) {
     gap: 1rem;
     display: grid;
-    grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
```

Comment:
My CSS isn't the best but I worry about changing this from the fraction to something more rigid. Might that cause issues art different screen sizes?

Reply:
I test a bit with different resolutions and in mobile view on the browser and in theory not. 

I changed that becase the 1fr make the cards wider when you filter them and leave for example 1 or 2 cards only 😅

My css isnt also the best so any suggestion to maintain the cards with the same width at filter will be apreciatted